public/style.css

```
@@ -1,35 +1,34 @@
-
-.buttons{
+.buttons {
   display: inline-block;
   margin: 0px 10px;
 }
 
-.input-form{
+.input-form {
   display: flex;
   margin-bottom: 10px; 
   width: 50%; 
```

行末に余計なスペースが含まれています。他にもあるので確認して削除してください。

```suggestion
  margin-bottom: 10px;
  width: 50%;
```

---

確認して削除しました。